pkg/quota/interfaces.go

```diff
@@ -51,6 +51,8 @@ type Evaluator interface {
 	Handles(operation admission.Attributes) bool
 	// Matches returns true if the specified quota matches the input item
 	Matches(resourceQuota *api.ResourceQuota, item runtime.Object) (bool, error)
+	// MatchingScopes takes the input specified list of scopes and pod object. Returns the set of scopes pod matches.
```

Comment:
nit: MatchingScopes takes the input specified list of scopes and and returns the set of scopes that matches the input object.